### PR TITLE
Implement gRPC SubscribeRawBlocks() RPC call

### PR DESCRIPTION
**Pull request by acidsploit**
Implementation of raw block subscription. It is essentially the same as the SubscribeBlocks logic,it just sends a serialized block.

Alternative option: Maybe this can instead be implemented in the original SubscribeBlocks() RPC as an option in the SubscribeBlocksRequest to specify how you want the BlockNotification to send you the block, serialized or deserialized.

modified:   bchrpc/bchrpc.proto	
modified:   bchrpc/pb/bchrpc.pb.go
modified:   bchrpc/server.go

### CODE REVIEW

File: bchrpc/bchrpc.proto [hunk 1]
```diff
@@ -280,6 +286,16 @@ message BlockNotification {
...
 }
 
+message RawBlockNotification {
+    enum Type {
+        CONNECTED = 0;
+        DISCONNECTED = 1;
+    }
+
+    Type type = 1;
+    RawBlock block = 2;
```
Comment (cpacia) [inline bchrpc/bchrpc.proto]: I think I would just make this `bytes` rather than nesting it in another object here.

**Comment by zquestz** [comment]
We should also make sure to update the docs with this new endpoint!

File: bchrpc/bchrpc.proto (hunk 1 above)
Comment (acidsploit) [reply bchrpc/bchrpc.proto]: Yes, looks much better that way indeed. Thanks! :)

**Comment by zquestz** [comment]
So I assume we are abandoning this PR in favor of https://github.com/gcash/bchd/pull/263 ?

**Comment by acidsploit** [comment]
I kept this one open because the other PR breaks backwards compatibility for SubscribeBlocks()

**Pull request closed without merging**